src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -158,17 +140,18 @@ else if (file.exists())
     }
 
     /**
-     * A mix of getCanonicalFile and getAbsoluteFile that returns the
-     * absolute path to the file without deferencing symbolic links.
+     * Get an absolute and partially simplified path to the given file.
+     *
+     * This differs from {@link File#getAbsoluteFile()} because it removes extraneous "." in the file path,
+     *   ex:    /path/./to/./directory/. -> /path/to/directory/
+     *
+     * It differs from {@link File#getCanonicalFile()} in that it doesn't dereference symbolic links or completely
+     * simplify the file path.  For instance ".." is not simplified by this method.
      *
      * @param file the file.
-     * @return the absolute path to the file.
+     * @return the absolute path to the file with redundant "./" removed.
      */
     public static File absolute(File file) {
```

Comment:
Can you either: find a better name for this method, or just use `getCanonicalFile()` directly (if you're uncomfortable with the latter option, try asking @kshakir about the genesis of this method in GATK3)

Reply:
getAbsoluteAndPartiallyCanonicalizedFile?

Reply:
Ah.  That makes sense.  paths.normalize seems like a better plan.

Reply:
absolute -> Paths.normalize()